### ISSUE TITLE
[Bug]: Error Module '"prosemirror-commands"' has no exported member 'type'. for Production

### ISSUE THREAD

**Issue by eviv3k** [sibiraj-s/ngx-editor#404]
### What happened?

On production, we are facing the error Module '"prosemirror-commands"' has no exported member 'type'.

I have commented on the PR code: https://github.com/sibiraj-s/ngx-editor/commit/9e7f1ad9d58654f048962df2e82ee941b3747d5c

### Version

v12.4.1

### Angular Version

v13.3.

### Relevant log output

```shell
Error: node_modules/@ng-bootstrap/ng-bootstrap/util/positioning.d.ts:1:55 - error TS2307: Cannot find module '@popperjs/core' or its corresponding type declarations.

1 import { Placement as PopperPlacement, Options } from '@popperjs/core';
                                                        ~~~~~~~~~~~~~~~~


Error: node_modules/ngx-editor/lib/commands/Heading.d.ts:2:10 - error TS2305: Module '"prosemirror-commands"' has no exported member 'type'.       

2 import { type Command } from 'prosemirror-commands';
           ~~~~


Error: node_modules/ngx-editor/lib/commands/Heading.d.ts:2:15 - error TS1005: ',' expected.

2 import { type Command } from 'prosemirror-commands';
                ~~~~~~~
                ~~~~~~~~~~~


Error: node_modules/ngx-editor/lib/commands/Link.d.ts:2:10 - error TS2305: Module '"prosemirror-commands"' has no exported member 'type'.

2 import { type Command } from 'prosemirror-commands';
           ~~~~


Error: node_modules/ngx-editor/lib/commands/Link.d.ts:2:15 - error TS1005: ',' expected.

2 import { type Command } from 'prosemirror-commands';
```
```


### Willing to submit a PR?

_No response_

**Comment by sibiraj-s** [comment]
Strange. Works fine for me. Do you have any @types/prosemirror-* packages installed locally?

**Comment by eviv3k** [comment]
Oh, thank you!

I just have installed `@types/prosemirror-commands` (`npm i @types/prosemirror-commands --save-dev`)

**Comment by sibiraj-s** [comment]
Glad the issue is fixed. But you don't have to install it manually, since its already a dependency of the package. https://github.com/sibiraj-s/ngx-editor/blob/master/projects/ngx-editor/package.json#L26

**Comment by ShridharGowda** [comment]
Issue resolved with version 13.0.0. Ohh man.. I struggled like anything with this issue.

**Comment by blerinaademi** [comment]
I have updated ngx-editor  to "ngx-editor": "^13.0.0", but now I have still those errors


Error: node_modules/ngx-editor/lib/commands/Image.d.ts:1:15 - error TS1005: ',' expected.

1 import { type EditorState, type Command } from 'prosemirror-state';
                ~~~~~~~~~~~
node_modules/ngx-editor/lib/commands/Image.d.ts:1:33 - error TS1005: ',' expected.

1 import { type EditorState, type Command } from 'prosemirror-state';
                                  ~~~~~~~
